nox.py

```diff
@@ -30,13 +30,13 @@ def docs(session):
     # Install Sphinx and also all of the google-cloud-* packages.
     session.chdir(os.path.realpath(os.path.dirname(__file__)))
     session.install('Sphinx >= 1.6.2', 'sphinx_rtd_theme')
+    session.install('-e', '.')
```

Comment:
1. This seems wrong: it'll pull **all** of our sub-packages from GitHub first
1. Can you drop this change from this PR